Update restore-sql-pool-from-deleted-workspace.md

azurepocmain opened the pull request:
I updated the below as the desired date time selection should not be an approximate timeframe it needs to be explicit, or it will fail. Please feel free to reach out to me for any questions. 


# Define the approximate point in time the workspace was dropped as DroppedDateTime "yyyy-MM-ddThh:mm:ssZ" (ex. 2022-01-01T16:15:00Z) $PointInTime="<DroppedDateTime>" 
$DroppedDateTime = Get-Date -Date $PointInTime

azurepocmain committed:
Update restore-sql-pool-from-deleted-workspace.md

I updated the below as the desired date time selection should not be an approximate timeframe it needs to be explicit, or it will fail. 


# Define the approximate point in time the workspace was dropped as DroppedDateTime "yyyy-MM-ddThh:mm:ssZ" (ex. 2022-01-01T16:15:00Z)
$PointInTime="<DroppedDateTime>" 
$DroppedDateTime = Get-Date -Date $PointInTime

WilliamDAssafMSFT commented on articles/synapse-analytics/backuprestore/restore-sql-pool-from-deleted-workspace.md:
```diff
@@ -50,11 +50,11 @@ In this article, you learn how to restore a dedicated SQL pool in Azure Synapse
+    Get-AzSynapseSqlPoolRestorePoint -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Name $SQLPoolName
+    # Pick the desired restore point after verifying the date time output from the above command. Add it to the where predicate in the below script where x/xx/xxxx is seen.
+    $PointInTime = Get-AzSynapseSqlPoolRestorePoint -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Name $SQLPoolName | Where-Object "RestorePointCreationDate" -eq "x/xx/xxxx x:xx:xx xM" | Select-Object -ExpandProperty RestorePointCreationDate
+  
```
@azurepocmain 
This appears to change the $pointInTime variable from the dropped date of the workspace, to the restore point at "x/xx/xxxx x:xx:xx xM". That right?
The later script sample uses `$DroppedDateTime` variable as well. Please update either this code sample or the later one.

WilliamDAssafMSFT replied:
By specifying "x/xx/xxxx x:xx:xx xM" we are still asking customers to bullseye the exact point in time the workspace was dropped. Have we solved the problem?

azurepocmain replied:
Thanks @WilliamDAssafMSFT good catch it should have been $DroppedDateTime I have since made the correction. actually I have the incorrect command it should be  Get-AzSynapseDroppedSqlPool apologies I was conducting two test let me update that and provide an update here once compelted.

azurepocmain replied:
Hi @WilliamDAssafMSFT tested and validated all the commands. I used Get-AzSynapseDroppedSqlPool  with the client to get the exact drop date to use for the restore. I added that if the workspace is dropped, they can easily recreate it with the same values and the command will work. Tested that as well. Overall, the invocation of "$DroppedDateTime = Get-AzSynapseDroppedSqlPool -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Name $DatabaseName | Select-Object -ExpandProperty DeletionDate" which should only have one value should prevent the errors that the client experienced when they provided an estimated date time prior to preform the deleted database restore.

azurepocmain committed:
Update restore-sql-pool-from-deleted-workspace.md

Alter the variable $PointInTime to $DroppedDateTime so that it is in the subsequent commands as $PointInTime is not the correct variable to use it should have been $DroppedDateTime.

azurepocmain committed:
Update restore-sql-pool-from-deleted-workspace.md

Correct the script to provide the correct code cmdlet that I used with the client which was Get-AzSynapseDroppedSqlPool. Added notes regarding that the Get-AzSynapseDroppedSqlPool needs to have the workspace available. If not, they can easily recreate the same workspace with the same values and the command will pick up the dropped database to be safely restored.
